### DISCUSSION TITLE
Scaling - How to?

### DISCUSSION THREAD

**Discussion by s256** [utmstack/UTMStack#450] [Q&A] [answered]
Hi all,
how would one configure the horizontal scaling with the "Slave" nodes?
Is it only for opensearch & logstash? Or does the backend also has multiple workers?

And: I'm always trying to run the database on the dedicated node and not on the same machine. 
Same would make sense for opensearch (read: run a dedicated opensearch node/cluster).

Separation of concerns basically on an infra level.

Coming from other XDRs, this one seems like a nice alternative.

**Comment by osmontero** [comment, marked as answer]
Hello! Thanks for your question. We have opened an issue to add this feature.

https://github.com/utmstack/UTMStack/issues/455
